packages/shorebird_cli/lib/src/commands/patch_command.dart

```diff
@@ -94,6 +106,14 @@ class PatchCommand extends ShorebirdCommand
       return ExitCode.software.code;
     }
 
+    final force = results['force'] == true;
```

Comment:
Looks fine, but isn't this also solved by adding default: false to the addFlag?

Reply:
I tried this – `results['force']` is still null if no arg is provided.